### DISCUSSION TITLE
base64 assets dumps are disturbing in stdout, better stdout log possibility?

### DISCUSSION THREAD

**Discussion by MirKml** [webpack/webpack#14792] [Q&A] [answered]
Hello, webpack now compiles images as assets, it's fine, no external loader is necessary.
When it generates base64 codes for images, it writes these into stdout when builds bundle, which invalidates other information which webpack produces to stdout, because these base64 codes are so huge. It's only presented in development mode.

output example:
```
modules by path data:image/ 41.4 KiB
  data:image/png;base64,iVBOR........
 ......tons of lines with base64
 ......tons of lines with base64
  ..... rkJggg== 35 KiB [built] [code generated]
 ```
  
old file/url-loaders produces much useful output
 ```
   asset modules 1.55 KiB (javascript) 1.8 MiB (asset)
    modules by path ../node_modules/..../assets/*.jpg 84 bytes (javascript) 1.73 MiB (asset)
 ```
  
It is possible with new built-in assets management get better output for development build, then huge base64 dumps for converted assets?

**Comment by alexander-akait** [comment, marked as answer]
Yes, make sense, feel free to open an issue
